pkgs/development/python-modules/linuxpy/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pytestCheckHook,
+  python312Packages,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "linuxpy";
+  version = "0.20.0";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-mNWmzl52GEZUEL3q8cP59qxMduG1ijgsvGoD5ddSG94=";
+  };
+
+  pythonImportsCheck = [ "linuxpy" ];
+
+  # Checks depend on WARD testing framework which is broken
+  doCheck = false;
+  nativeCheckInputs = [
+    pytestCheckHook
+    python312Packages.ward
+  ];
+
+  meta = with lib; {
+    description = "Human friendly interface to linux subsystems using python";
+    homepage = "https://github.com/tiagocoutinho/linuxpy";
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```
https://github.com/tiagocoutinho/linuxpy/blob/master/pyproject.toml#L10